AutoRest/Generators/CSharp/CSharp/Templates/MethodTemplate.cshtml

```diff
@@ -116,7 +126,7 @@ public async Task<@(Model.OperationResponseReturnTypeString)> @(Model.Name)WithH
     {
         <text>
     // Serialize Request
-    string requestContent = JsonConvert.SerializeObject(@(Model.RequestBody.Name), @(Model.GetSerializationSettingsReference(Model.RequestBody.Type)));
+    string requestContent = JsonConvert.SerializeObject(@(Model.RequestBody.ParameterAccessor), @(Model.GetSerializationSettingsReference(Model.RequestBody.Type)));
```

Comment:
@yugangw-msft Yes, because we allow you to group requestBody into the parameterGroup as well, which means that instead of referring to it by "body" it'll be "parameters.Body" (or something like that).  Take a look at the generated code `ParameterGroupingOperations.cs` and `ParameterGroupingPostRequiredParameters.cs` to see what I mean.